nixos/modules/services/audio/mpdscheduler.nix

```diff
@@ -0,0 +1,59 @@
+{ config, lib, pkgs, ... }:
+with lib;
+let 
+  cfg = config.services.mpdscheduler;
+in {
+  options.services.mpdscheduler = {
+    enable = mkEnableOption "mpdscheduler service";
+    host = mkOption {
+      type = types.str;
+      description = "the mpd host to connect to";
+      default = "localhost";
+      example = "mpdHost.local";
+    };
+    port = mkOption {
+      type = types.int;
+      description = "the mpd tcp port to connect to";
+      default = 6600;
+    };
+    additionalChannels = mkOption {
+      type = types.bool;
+      default = true;
+      description = "whether the 'alarm', 'sleep' and 'cancel' channels shall be used in addition to the 'scheduler' channel";
+    };
+    fadeTime = mkOption {
+      type = types.int;
+      default = 30;
+      description = "alarm/sleep fade time in seconds";
+      example = 15;
+    };
+    maxVolume = mkOption {
+      type = types.int;
+      default = 100;
+      description = "the volume to wake at in percent";
+      example = 50;
+    };
+    waitForMpd = mkOption {
+      type = types.bool;
+      default = true;
+      description = "whether to wait for mpd to start before starting, set to false if mpdscheduler is connecting to mpd on another machine";
+    };
+  };
+
+  config = mkIf cfg.enable {
+    systemd.services.mpdscheduler = {
+      wantedBy = [ "multi-user.target" ];
+      environment = {
```

Comment:
Are there any other options? I'm guessing this only takes environment variables with no config file?

Reply:
Yes.